packages/grafana-data/src/transformations/transformers/joinDataFrames.ts

```diff
@@ -272,16 +277,29 @@ function nullExpand(yVals: Array<number | null>, nullIdxs: number[], alignedLen:
 }
 
 // nullModes is a tables-matched array indicating how to treat nulls in each series
-export function join(tables: AlignedData[], nullModes?: number[][]) {
+export function join(tables: AlignedData[], nullModes?: number[][], innerJoin?: boolean) {
```

Comment:
```suggestion
export function join(tables: AlignedData[], nullModes?: number[][], innerJoin = false) {
```

Reply:
Resolving conversation - implemented using the suggestion by @ryantxu to use a mode